src/test/java/core/basesyntax/service/WriterServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.WriterServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceTest {
+    private static final String VALID_FILE_NAME = "src/test/resources/report.csv";
+    private static final String INVALID_FILE_NAME = "src/tests/resources/report.csv";
```

Comment:
maybe you forgot to change path?

Reply:
Hi. No )) Look valid file is in the test directory and the invalid one is in non existing "tests" directory. But I think it would be better to change this path to another one just to make it clearer for another developer =)

Reply:
done